### DISCUSSION TITLE
Profile is floating on top

### DISCUSSION THREAD

**Discussion by Mingyu-Song** [community/community#19589] [Projects and Issues] [answered]
https://user-images.githubusercontent.com/61491189/176113538-30627db2-545d-4db7-837c-de85c3a0b634.mov

My profile is floating on the top left, is this normal?

**Comment by D7EAD** [comment]
Mine is as well! How odd, I didn't even notice! I don't remember this being by design...

**Comment by melvincwng** [comment, marked as answer]
@Mingyu-Song, thanks for highlighting this issue to the community.

Can you kindly report this bug to [GitHub Bug Report](https://support.github.com/contact/bug-report) over here?

Hope this helps.

**Reply by melvincwng** [reply]
@Mingyu-Song, if this answer is helpful, I would appreciate it if you could do the following:

1. 'Mark as answer'
2. Follow me on GitHub

Do ping me or tag me here if you face any other issues or have any queries.

Thank you, and have a nice day ahead 😄 

감사합니다친구^^

**Reply by Mingyu-Song** [reply]
Oh, thank you. I didn't know there was a bug report.

**Reply by melvincwng** [reply]
@Mingyu-Song, pleasure in helping ^^. 

Let us know if you encounter any other issues, in the future.

Take care.
